src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -273,30 +274,34 @@ export function* lcManager(params={}) {
 }
 
 
-function updateRawTableChart(timeCName, fluxCName, mission='') {
+function updateRawTableChart(timeCName, fluxCName, converterId) {
     var chartX = get(getChartDataElement(LC.RAW_TABLE), ['options', 'x', 'columnOrExpr']);
     var chartY = get(getChartDataElement(LC.RAW_TABLE), ['options', 'y', 'columnOrExpr']);
 
     if (chartX === timeCName && chartY === fluxCName) return;
 
     if (timeCName && fluxCName) {
 
-        const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const rawTable = getTblById(LC.RAW_TABLE);
+
+        //const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const title = getConverter(converterId).showPlotTitle? rawTable.request.META_INFO.title :'';
```

Comment:
For the records, we discussed that earlier today. 
I agree with Loi. 
We agree that Lijun will change and add getPlotTitle callback in the converter so any mission can pass in any plot title.